3_javascript/javascript1/week1/script2.js

```diff
@@ -0,0 +1,11 @@
+let firstWords = ["Eazy", "Awesome", "Corporate", "Innovative", "Global", "Dynamic", "Text", "Creative", "Fusion", "Spark"];
+let secondWords = ["Tech", "Solutions", "Labs", "Innovations", "Hub", "Network", "Edge", "Forge", "Nest", "Wave"];
+
+let randomIndexFirst = Math.floor(Math.random() * 10);
+let randomIndexSecond = Math.floor(Math.random() * 10);
+
+let startupName = firstWords[randomIndexFirst] + " " + secondWords[randomIndexSecond];
+let nameLength = startupName.length;
```

Comment:
If you want to know the exact number of characters, you can use`let nameLength = startupName.length - 1` . Because you are counting with the empty space between two words.